src/components/dialogs/parameters/shortcircuit/short-circuit-tension-table.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FunctionComponent, useCallback, useMemo } from 'react';
+import { Table, TableBody, TableCell, TableRow } from '@mui/material';
+import { useIntl } from 'react-intl';
+
+interface TensionTableProps {
+    isNominal: boolean;
+}
+const TensionTable: FunctionComponent<TensionTableProps> = ({ isNominal }) => {
+    const intl = useIntl();
+
+    const createRow = useCallback(
+        (
+            name: string,
+            v1: number,
+            v2: number,
+            v3: number,
+            v4: number,
+            v5: number,
+            v6: number,
+            v7: number
+        ) => {
+            return { name, v1, v2, v3, v4, v5, v6, v7 };
+        },
+        []
+    );
+
+    const rows = useMemo(() => {
```

Comment:
```suggestion
        const rows = useMemo(
        () => [
            {
                name: intl.formatMessage({ id: 'shortCircuitNominalVoltage' }),
                values: [380, 225, 150, 90, 63, 45, 20],
            },
            {
                name: intl.formatMessage({ id: 'shortCircuitInitialTension' }),
                values: isNominal
                    ? [400, 225, 150, 90, 63, 45, 20]
                    : [420, 245, 165, 99, 69.3, 49.5, 22],
            },
        ],
        [isNominal, intl]
    );
```

this way yo ucan avoir creating "createRow" function and importing useCallback

Reply:
I thought  of creating a function which takes care of the creation of rows and  to centralize future changes.
But since it hardly change it' good to go.Thanks.